### ISSUE TITLE
Can't Compile in Xcode 

### ISSUE THREAD

**Issue by bwalls** [brockgr/websaver#21]
Downloaded project, but when trying to compile I get a bunch of errors like:
No visible @interface for 'WKWebView' declareds the selector 'whatever'

Trying to compile using the current Xcode on 10.10.5.

**Comment by uahengojr** [comment]
Yeah, I'm having the same issue. Has anyone found a solution to this?

**Comment by MonsterCritic** [comment]
Please, fix!

**Comment by emrehayta** [comment]
Has anyone a solution for this issue?

**Comment by wieland-s** [comment]
Stil have this problem :(

**Comment by TomasHubelbauer** [comment]
Are you folks still experiencing issues? There was a recent PR merged that brought up the support to newer macOS versions and cleaned up some issues. It is possible that this will now work for you.
